### PR TITLE
Closes #2296 - add ownerLongName during update

### LINKED ISSUE #2296: OwnerLongName is not set during update
Todos:
- Add tests
- Set ownerLongName during update

### PR DESCRIPTION
**Pull request by ryzheboka**
https://sonarcloud.io/summary/new_code?id=ryzheboka_taskana&branch=TSK-2296

---
Release Notes:

```

```
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] After integration of the PR, I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → (Closes) #<Issue Number>: Your commit message. i.e. Closes #2271: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: lib/taskana-core-test/src/test/java/acceptance/task/update/UpdateTaskAccTest.java [hunk 1]
```diff
@@ -569,19 +573,38 @@ private ObjectReferenceImpl createObjectReference(
...
+          .longName("Ella Mustermann")
+          .buildAndStore(userService, "businessadmin");
     }
 
     @WithAccessId(user = "user-1-2")
     @Test
     void should_AllowTimestampServiceLevelMismatch_When_ConfigurationAllowsIt() throws Exception {
       Task task =
           TaskBuilder.newTask()
+              .owner("user-1-2")
```
Comment (gitgoodjhe) [inline lib/taskana-core-test/src/test/java/acceptance/task/update/UpdateTaskAccTest.java:607]: why did you add this in this test?

**Review by gitgoodjhe — CHANGES_REQUESTED** [review]

File: lib/taskana-core-test/src/test/java/acceptance/task/update/UpdateTaskAccTest.java (hunk 1 above)
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/task/update/UpdateTaskAccTest.java:607]: It was a copy-paste error from trying out before writing tests. I will remove it :)

**Review by gitgoodjhe — APPROVED** [review]
